karoo_gp/pause.py

```diff
@@ -243,14 +243,14 @@ def pause(menu_dict):
                     query = input('\n\t Select a Tree to evaluate: ')
                     if query == '':
                         break
-                    elif int(query) in list(range(1, menu_dict['pop_b_len'])):
+                    elif int(query) in range(menu_dict['pop_b_len']):
```

Comment:
I think we covered this in GChat:
- Tree id's start from 1
- In a population, they're stored in a list with normal (from 0) indexing
- Whenever possible I use `tree.id`, which is accurate
- Otherwise, if indexing, use +1/-1
This (above) is the odd case of `population_b`, whose `tree.id` is also incorrect, because it's part of an evolving population and has the id of it's parent(s). So I use index +1 here as well.